assets/queries/dockerCompose/cgroup_not_default/metadata.json

```
@@ -0,0 +1,10 @@
+{
+  "id": "4d9f44c6-2f4a-4317-9bb5-267adbea0232",
+  "queryName": "Cgroup Not Default",
```

```suggestion
  "queryName": "Cgroup Undefined",
```

Please, also update the query folder name